public_html/gateway.py

```diff
@@ -19,14 +19,14 @@
 
 PROVIDERS = {
     'ARIN': 'https://whois.arin.net/rest/ip/{0}',
-    'RIPENCC': 'https://apps.db.ripe.net/search/query.html?searchtext={0}#resultsAnchor',
-    'AFRINIC': 'http://afrinic.net/cgi-bin/whois?searchtext={0}',
+    'RIPENCC': 'https://apps.db.ripe.net/db-web-ui/query?searchtext={0}',
+    'AFRINIC': 'https://www.afrinic.net/whois-web/public/?lang=en',
```

Comment:
I believe the user is supposed to paste the IP into the AFRINIC form manually with this.  I'd like to see the instruction somewhere explicitly. Maybe in the link anchor or in the tooltip. We might want to skip AFRINIC in the "Location:" redirection, too.

Reply:
Changed it to RDAP, provides the same information, just ugly.